extension/src/common/colors.ts

```diff
@@ -0,0 +1,41 @@
+const colorsList = [
+  '#945dd6',
+  '#13adc7',
+  '#f46837',
+  '#48bb78',
+  '#4299e1',
+  '#ed8936',
+  '#f56565'
+] as const
+
+export type Color = (typeof colorsList)[number]
+
+export const copyOriginalColors = (): Color[] => [...colorsList]
+
+const boundingBoxColorsList = [
```

Comment:
Apologies, I thought we agreed that we don't want the experiment colors (`colorsList`) to be used to represent a bounding box class, because it might seem if that class is related to a particular selected experiment? 

Did I misunderstand your previous comment? If you would like us to use the experiment colors as possible bounding box class colors, I'm happy to update :) 

Or are you meaning something else entirely that I'm missing?

Reply:
You're right, that makes sense. Sorry, I forgot about that fact.